tomplot/__init__.py

```diff
@@ -7,3 +7,4 @@
 from tomplot.quiver_plot import *                                         # noqa
 from tomplot.regridding import *                                          # noqa
 from tomplot.tomplot_tools import *                                       # noqa
+from tomplot.local_area_plot import *                                     # noqa
```

Comment:
I am wondering if we want to actually call this module `data_manipulation`. We could eventually move the `reshape_data` routines into this module as well.

A petty point though: I think in the `__init__`, things are alphabetised, so whatever we call this module, can you keep this list in alphabetical order?